README.md

```diff
@@ -73,27 +71,27 @@ Before pushing you may run build to ensure your JS/CSS is built.
 npm run-script build
 ```
 
-Alternatively if you have `gulp` installed globally, you can also simply run `gulp`.
+Alternatively if you have `gulp` installed globally, simply run `gulp`.
 
 ### Running unit-tests locally
 
 Make sure to include the `tests/phpunit.xml` as an *alternative configuration file* in your IDE.
 
-Alternatively all tests can be run, by executing
+Alternatively run all tests with
 
 ```
 composer test
 ```
 
 ## Deployment & Release
-1. Update the German and English changelogs in folder `meta/documents`.
+1. Update the german and english change logs in folder `meta/documents`.
```

Comment:
Countries, nationalities, and languages may always be capitalized.

Reply:
Correct please change back.